README.md

```diff
@@ -1,12 +1,14 @@
 # Health-Barbecue
 
-## Pre-requirement
+## Requirements
 
 To launch the global application, you need:
-- docker [https://www.docker.com/](https://www.docker.com/)
-- docker-compose [https://docs.docker.com/compose/install/](https://docs.docker.com/compose/install/)
+- [Docker](https://www.docker.com/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
 
-## Requirements on macOS
+### Requirements on macOS
+
+On macOS, you'll need specific steps to install Docker, Docker Compose, and the .NET Core SDK.
```

Comment:
On macOS, specific steps are needed to install Docker, Docker Compose, and the .NET Core SDK.